python/replicate/config.py

```diff
@@ -20,11 +20,13 @@ def load_config(project_dir: str) -> Dict[str, Any]:
     """
     Loads config from directory
     """
-    #  Support replicate.yml too: https://github.com/replicate/replicate/issues/351
-    try:
+    if os.path.isfile(os.path.join(project_dir, "replicate.yaml")):
```

Comment:
Maybe this could be turned into a for loop over `["replicate.yaml", "replicate.yml"]`?

Reply:
Done